scripts/build-binary.sh

```diff
@@ -6,7 +6,14 @@ source ./scripts/test_lib.sh
 
 VER=$1
 REPOSITORY="${REPOSITORY:-git@github.com:etcd-io/etcd.git}"
-
+# When ${CURRENT_BRANCH} is 1, the scripts doesn't clone the remote
+# repository at all, nor checkout the branch; instead, it just builds
+# the binaries in current branch in local existing directory. The ${VER}
+# will be ignored as well in such case.
+# Otherwise if ${CURRENT_BRANCH} is 0, it clones the remote repository
+# firstly, then checkout the target tagged version, and finally build
+# binaries.
```

Comment:
Explaining the flag is just a consequence of a bad name of variable. Please prefer more correct naming scheme. Like in above comment this feature sounds more like `--local`

Reply:
see above.